app/main.py

```diff
@@ -1,13 +1,46 @@
 def format_linter_error(error: dict) -> dict:
-    # white your code here
-    pass
+    return {
+        "line": error.get("line_number"),
+        "column": error.get("column_number"),
+        "message": error.get("text"),
+        "name": error.get("code"),
+        "source": "flake8"
+    }
 
 
 def format_single_linter_file(file_path: str, errors: list) -> dict:
-    # white your code here
-    pass
+    return {
+        "errors": [
+            {
+                "line": error.get("line_number"),
+                "column": error.get("column_number"),
+                "message": error.get("text"),
+                "name": error.get("code"),
+                "source": "flake8"
+            }
+            for error in errors
+        ],
+        "path": file_path,
+        "status": "passed" if "errors" == [] else "failed"
+
```

Comment:
You have implemented `format_linter_error` above, why did you duplicate code from that function?

Reply:
Thanks! I have understood.